libnavui-dropin/src/main/res/layout/mapbox_info_panel_active_guidance_layout.xml

```diff
@@ -4,43 +4,51 @@
     xmlns:tools="http://schemas.android.com/tools"
     android:layout_width="match_parent"
     android:layout_height="match_parent"
-    tools:parentTag="android.widget.FrameLayout"
-    >
+    tools:background="@drawable/mapbox_bg_info_panel"
+    tools:parentTag="android.widget.FrameLayout">
 
-    <androidx.constraintlayout.widget.ConstraintLayout
+    <LinearLayout
+        android:orientation="vertical"
         android:layout_width="match_parent"
-        android:layout_height="wrap_content"
-        android:padding="8dp"
-        >
+        android:layout_height="wrap_content">
 
-        <com.mapbox.navigation.ui.tripprogress.view.MapboxTripProgressView
-            android:id="@+id/tripProgressView"
-            android:layout_width="wrap_content"
-            android:layout_height="wrap_content"
-            app:layout_constraintEnd_toEndOf="parent"
-            app:layout_constraintTop_toTopOf="parent"
-            app:layout_constraintStart_toStartOf="parent" />
+        <androidx.constraintlayout.widget.ConstraintLayout
+            android:layout_width="match_parent"
+            android:layout_height="@dimen/mapbox_infoPanel_peekHeight">
+
+            <View
+                style="@style/MapboxStyle.DropIn.DragHandle"
+                android:id="@+id/handle"
+                app:layout_constraintTop_toTopOf="parent"
+                app:layout_constraintEnd_toEndOf="parent"
+                app:layout_constraintStart_toStartOf="parent" />
+
+            <include layout="@layout/mapbox_partial_trip_progress"
```

Comment:
Let's remove `mapbox_partial_trip_progress` and `mapbox_partial_trip_progress_custom` and use `MapboxTripProgressView` here instead of `include`.